source/connection-strings/mongodb-database-connection-strings.txt

```diff
@@ -1,3 +1,126 @@
+Identifier (URI) formats for defining connections to your target 
+MongoDB database. 
+
+Relational Migrator can use both Atlas and On-Premises URIs. This page 
+provides separate instructions for each deployment type.
+
+Atlas
+-----
+
+Connect to Your Deployment
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To provision user accounts in Atlas you need to be 
+`logged in to the Atlas GUI <https://cloud.mongodb.com/v2/>`__.
+
+Create a New User
+~~~~~~~~~~~~~~~~~
+
+#. In the :guilabel:`Security` section of the left navigation, click :guilabel:`Database Access`.
+#. Click :guilabel:`Add New Database User`.
+#. In the Authentication Method section of the :guilabel:`Add New Database User` modal window, select the box labeled :guilabel:`Password`.
+#. Under :guilabel:`Password Authentication`, enter the username ``migrator-service`` for the new user in the top text field.
+#. Enter the password ``password`` for the new user in the lower text field.
+#. Under :guilabel:`Database User Privileges`, click :guilabel:`Built-in Role`.
+#. Select :guilabel:`Read and write to any database`.
+#. Click :guilabel:`Add User`
+
+Connection URI
+~~~~~~~~~~~~~~
+
+Using the previously created account credentials, format the connection
+URI for the target database. In this case ``MongoEnterprises``.
+
+.. code-block::
+
+   mongodb+srv://migrator-service:password@sandbox.xxxxx.mongodb.net/MongoEnterprises
+
+On-Premises
+-----------
+
+In this example, use ``mongosh`` to provision a user account that 
+connects to the ``MongoEnterprises`` database.
+
+This examples assume your deployment is running on ``localhost`` and the 
+default port of ``27017``.
+
+Connect to Your Deployment
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When connecting to your deployment with ``mongosh``, use a user account 
+which has the `createRole <https://www.mongodb.com/docs/manual/reference/method/db.createRole/>`__
+permissions in both the ``admin`` and ``MongoEnterprises`` databases.
+
+Copy the following code with your admin credentials into a terminal to 
+connect to your deployment with ``mongosh``.
+
+.. code-block::
+
+   mongosh "mongodb://myadminuser:myadminpassword@localhost:27017/admin"
+
+For details on ``mongosh`` connections see : 
+`Connecting to your MongoDB Deployment <https://www.mongodb.com/docs/mongodb-shell/connect/>`__.
+
+Create a new User
+~~~~~~~~~~~~~~~~~
+
+In ``mongosh`` run the following command to create a 
+user in the ``admin`` database:
+
+.. code-block::
+
+   use admin
+
+   db.createUser(
+      { user: "migrator-service",
+         pwd: "password",
+         roles:[{role: "readWrite" , db:"MongoEnterprises"}]
+      }
+   )
+
+Create the same user in the user in the ``MongoEnterprises`` database:
+
+.. code-block::
+
+   use MongoEnterprises
+   db.createUser(
+      { user: "migrator-service",
+         pwd: "password",
+         roles:[{role: "readWrite" , db:"MongoEnterprises"}]
+      }
+   )
+
+These commands:
+
```

Comment:
(typo)
s/b "Create a new user ..."
(both subject verb alignment and word mix up)